lite/examples/image_classification/raspberry_pi/requirements.txt

```diff
@@ -1,5 +1,5 @@
 argparse
 numpy>=1.20.0  # To ensure compatibility with OpenCV on Raspberry Pi.
 opencv-python~=4.5.3.56
-tflite-support==0.4.0
+tflite-support-nightly
```

Comment:
Please change to tflite-support>=0.4.2 and test if everything works fine. Do the same for other PRs.